types/accept-language-parser/index.d.ts

```diff
@@ -1,17 +1,18 @@
 // Type definitions for accept-language-parser 1.5
 // Project: https://github.com/opentable/accept-language-parser
 // Definitions by: Niklas Wulf <https://github.com/kampfgnom>
+//                 Wooram Jun <https://github.com/chatoo2412>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.1
+// TypeScript Version: 2.4
```

Comment:
Is there a particular reason you need to increase the minimum allowable TS version? Sorry if I'm missing something obvious.

Reply:
The tester said so. I think it's because of string enums in the test file. But string enums are not related with `index.d.ts` itself, what should I do?

Reply:
@chatoo2412 You could do something like:
```
type Lang = "en-US" | "ko-KR"
const EnUs: Lang = "en-Us";
const KoKr: Lang = "ko-KR";
```
I'd agree that's not as clean as the other, but it seems silly to exclude certain users just because of the test file.

Reply:
I agree with you, thanks for your solution. I'll update and force-push soon.